carbonmark/components/ViewWalletButton/styles.ts

```diff
@@ -0,0 +1,12 @@
+import { css } from "@emotion/css";
+
+export const viewWalletButton = css`
+  color: var(--manatee);
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  align-items: center;
+`;
+
+export const viewButton = css`
+  gap: 4px !important;
```

Comment:
Nit: use rem values

```gap: 0.4rem;```

Reply:
I actually took this from figma as-is. Is there anything to worry about when changing it from px to rem in this specific case?

Reply:
You can always double check to ensure the computed styles are the pixel value you are looking for in devtools.

For example - padding top & bottom here are set as 0.4rem.
<img width="456" alt="Screenshot 2023-07-22 at 19 17 51" src="https://github.com/KlimaDAO/klimadao/assets/92357475/27b1013e-2c0e-4d70-a672-18a7d0533bfc">

Switch from styles to computed and you can see that both values are 4px
<img width="455" alt="Screenshot 2023-07-22 at 19 18 11" src="https://github.com/KlimaDAO/klimadao/assets/92357475/497f04bc-f2ed-4f8b-8bd3-268a8eddde93">

Reply:
Thanks, I'll keep that in mind in the future. Had no idea about the computed tab. Done in [975daa4](https://github.com/KlimaDAO/klimadao/pull/1321/commits/975daa466b367c1a080f93693d7b5ccbcf2a8dc3)